requirements.txt

```diff
@@ -16,3 +16,4 @@ pytest==2.9.1
 requests==2.9.1
 six==1.10.0
 Werkzeug==0.11.8
+beautifulsoup4
```

Comment:
Though it is debatable, keeping all the indirect dependencies in requirements.txt with their version numbers makes sure the builds are repeatable. If a module is dependent on five others, and one of them gets an upgrade in a way that is not backward-compatible, that'd be a problem for us! That's why versioning is important.